pkg/recovery/recovery.go

```diff
@@ -60,11 +89,11 @@ func CrashLog(r interface{}, cfgLog log.GlobalConfig) {
 func writeHeapProfile(path string) {
 	f, err := os.OpenFile(path, os.O_CREATE|os.O_RDWR, 0644)
 	if err != nil {
-		log.S().Fatalf("crashlog: open heap profile error: %v", err)
+		log.S().Errorf("crashlog: open heap profile error: %v", err)
```

Comment:
if not returned, pprof.WriteHeapProfile will be called

Reply:
add return

Reply:
add return